internal/pkg/utils/reflect.go

```diff
@@ -120,3 +120,18 @@ func ReadTaggedFields(dir, relPath, tag string, target interface{}, errPrefix st
 
 	return nil
 }
+
+func SetFields(fields []*StructField, data *orderedmap.OrderedMap, target interface{}) {
```

Comment:
Helper nad reflection, ... nieco podobne uz v tomto subore je, ... ale to sa odstrani v dalsom PR.